dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/http/EndpointRequest.java

```diff
@@ -51,6 +61,20 @@ public SSLSocketFactory getSSLSocketFactory() {
         }
         return null;
     }
+    
+    public void instantiateStorage(OidcClientConfig config, HttpServletRequest request, HttpServletResponse response) {
```

Comment:
Multiple clients can end up overriding each other's storage and storage type here from multiple requests. Perhaps this method can be at the client level to create a storage, but not save it. Need to look into this further.

Reply:
okay. I moved the storage instantiation to token response validator itself for now.